sqlalchemy_bigquery/base.py

```diff
@@ -655,25 +656,79 @@ def get_column_specification(self, column, **kwargs):
 
     def post_create_table(self, table):
         bq_opts = table.dialect_options["bigquery"]
-        opts = []
 
-        if ("description" in bq_opts) or table.comment:
-            description = process_string_literal(
-                bq_opts.get("description", table.comment)
+        partition_by_or_cluster_by_or_options = []
+
+        if "partitioning" in bq_opts:
+            partition_expr = bq_opts.get("partitioning")
+
+            assert isinstance(
+                partition_expr, str
+            ), f"bigquery_partitioning dialect option accepts only {str}, provided {repr(partition_expr)}"
+
+            partition_by_or_cluster_by_or_options.append(
+                f"PARTITION BY {partition_expr}"
             )
-            opts.append(f"description={description}")
 
-        if "friendly_name" in bq_opts:
-            opts.append(
-                "friendly_name={}".format(
-                    process_string_literal(bq_opts["friendly_name"])
-                )
+        if "clustering_fields" in bq_opts:
+            clustering_fields = bq_opts.get("clustering_fields")
+
+            assert isinstance(
```

Comment:
See note above regarding `assert`. Replace this and any other `assert` statements.